bin/test

```diff
@@ -12,5 +12,10 @@ export DEVELOPMENT=1
 # Create environment
 cp .env-dist .env
 
+# pull any missing images before running, so we can use
+# --parallel (may not be needed once https://github.com/docker/compose/issues/4556
+# fixed?)
+docker-compose pull --parallel
```

Comment:
What about pulling the images from circleci.yml instead?
This file seems to be a generic test script and could be used not only on CI.

Reply:
What do you think about just removing this script altogether and putting its logic in circle.yml? I'm not sure if it's actually helpful. Locally I have always run `make test`.

Reply:
:+1: to that. I always forget if this file is actually used in CI or not. Let's move it to the circleci logic. I too always use the make directives for local development